addons/point_of_sale/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 from . import models
 from . import controllers
+from . import data
```

Comment:
?

Reply:
I think this PR is still open for no good reason.